src/Maestro/tests/Maestro.ScenarioTests/ScenarioTests_GitHubFlow.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.DotNet.DarcLib;
+using Microsoft.DotNet.Maestro.Client.Models;
+using NUnit.Framework;
+
+namespace Maestro.ScenarioTests
+{
+    [TestFixture]
+    [Category("PostDeployment")]
+    public class ScenarioTests_GitHubFlow : MaestroScenarioTestBase
+    {
+        private TestParameters _parameters;
+        private EndToEndFlowLogic testLogic;
+
+        public ScenarioTests_GitHubFlow()
```

Comment:
Don't need this empty constructor, I think. Since there isn't anything in it. :-)